dotnet/src/SemanticKernel/Http/HttpClientFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Concurrent;
+using System.Net.Http;
+
+namespace Microsoft.SemanticKernel.Http;
```

Comment:
I like this namespace, should probably consolidate the Reliability namespace here since it is http specific, too.

nit: newline after namespace declaration